cmd/cartesi-rollups-cli/root/read/input/input.go

```diff
@@ -0,0 +1,53 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package input
+
+import (
+	"encoding/json"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/cartesi/rollups-node/internal/logger"
+	"github.com/cartesi/rollups-node/pkg/graphqlutil"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "input",
+	Short:   "Reads input from node GraphQL",
+	Example: examples,
+	Run:     run,
+}
+
+const examples = `# Read specific input from GraphQL:
+cartesi-rollups-cli read input --index 5 --graphqlEndpoint http://0.0.0.0:4000/graphql`
```

Comment:
We can remove the `--graphqlEndpoint` from the example because this is the default value. It does not add much, making it look like the variable has no default.